svgr-template.js

```diff
@@ -8,7 +8,11 @@ const nativeImports = `
 
 const webStyles = `
   const {className: classNameProp} = other
-  const className = shadow ? css({ filter: theme.shadows.drop }) : undefined
+  const className = css({
+     filter: shadow ? theme.shadows.drop : undefined,
+     minHeight: height,
```

Comment:
interesting! i like this solve - i was thinking to flex-shrink: 0 the hint container for the icon, but this ensures this never occurs in any flex 🚀